docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-15.rst

```diff
@@ -0,0 +1,174 @@
+Squeeze
+=======
+
```

Comment:
why do we require such version? Is there similar operation in frameworks like PyTorch, TF, ONNX? Can we refer a link to such operation?

Reply:
It's needed for alignment with torch, more details can be found in the internal ticket: 143585.